src/datastore_export/export_datastore_to_gcs_service.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import httplib
+import json
+import logging
+import time
+
+import webapp2
+from google.appengine.api import app_identity
+from google.appengine.api import urlfetch
+
+from src.configuration import configuration
+
+
+class ExportDatastoreToGCSException(Exception):
+    pass
+
+
+class ExportDatastoreToGCSService(webapp2.RequestHandler):
+    @classmethod
+    def invoke(cls, request, response):
+        access_token, _ = app_identity.get_access_token(
+            'https://www.googleapis.com/auth/datastore')
+        app_id = configuration.backup_project_id
+        url = 'https://datastore.googleapis.com/v1/projects/%s:export' % app_id
+
+        output_url_prefix = cls.get_output_url_prefix(request)
```

Comment:
We could use convention staging.project_id.appspot.com bucket, where lifecycle management is enabled. No need for configuration then